src/en/charms-offline-config.md

```diff
@@ -0,0 +1,214 @@
+Title: Configuring Juju for offline usage
+TODO:  for localhost cloud and controller being on the same host as the client, it's possible to use LXD remotely. worth mentioning?
+
+# Configuring Juju for offline usage
+
+*This page makes use of information presented in the
+[Offline mode strategies][charms-offline-strategies] document. Please see that
+page for a full understanding.*
+
+There are three Juju entities to take into account when configuring Juju in a
+network-restricted environment. Each one of these may require specific
+treatment depending on the backing cloud type and in what manner the local
+network is restricted. These are:
+
+ - client
+ - controller
+ - machines
+
+Where the *client* is the Juju client from whence `juju` commands are invoked;
+the *controller* is the Juju machine acting as controller; and the *machines*
+are the Juju workload machines that get created during charm deployment.
+
+These entities interact in the following way: the client is responsible for
+creating the controller, and then manages the controller via the latter's API.
+The controller, in turn, is responsible for accessing and passing all needed
+resources to the machines.
+
+There are a few exceptions to the above rule. When creating a controller the
+client needs to both access and then transfer both the 'juju-gui' charm and the
+Juju agent to the controller.
```

Comment:
perhaps Juju agent should be explained in the previous text about machines

Reply:
My plan, which backfired, was to have an explanation of agents on a separate page. Just a few seconds ago I did manage to get a few lines from the Juju team. Let's say I do it later and then link from this current page.

Reply:
I will add an abridged definition and link to the full one in a later PR.

Reply:
Done.